data/campaigns/Legend_of_Wesmere/scenarios/chapter4/16_The_Chief_Must_Die.cfg

```diff
@@ -247,8 +247,8 @@
         faction=Custom
         [ai]
             [stage]
-                engine=fai
-                name=unit_formulas
+                id=main_loop
```

Comment:
Hmm? Is this actually necessary? Does this end up adding the stage twice? I think the whole `[ai]` block could just be deleted, right?

Reply:
It does not add the stage twice.  The AI either adds the default stage, or whatever is given, not both.  The point here is that we want an empty AI stage, except for the patrol MAI (there are other units on this side that are supposed to do nothing).  So we can either add the default stage and delete every single CA, or add an empty stage.  I decided that the latter is easier and more robust to possible future changes to the default AI.

Reply:
Ah, okay, so this is basically equivalent to setting idle AI… actually… does that still exist?

Reply:
Yes, it still exists and is, in fact, used for some of the other sides in the same scenario.  However, it is somehow handled differently internally (I forgot the details) and trying to add the MAI to the Idle AI still results in idle behavior.  That's actually the first thing I tried.